--- qt-creator-1.0.0-src/src/app/main.cpp.orig	2009-03-05 14:48:59.398196319 +0100
+++ qt-creator-1.0.0-src/src/app/main.cpp	2009-03-05 14:53:50.581940902 +0100
@@ -172,6 +172,9 @@
 static inline QStringList getPluginPaths()
 {
     QStringList rc;
+    rc.push_back(QLatin1String("/usr/lib64/qtcreator/plugins"));
+    return rc;
+
     // Figure out root:  Up one from 'bin'
     QDir rootDir = QApplication::applicationDirPath();
     rootDir.cdUp();
