--- qt-creator-opensource-src-11.0.2/cmake/Findyaml-cpp.cmake.orig	2023-08-14 03:53:08.000000000 +0200
+++ qt-creator-opensource-src-11.0.2/cmake/Findyaml-cpp.cmake	2023-08-30 21:30:11.818177005 +0200
@@ -15,7 +15,7 @@ if (yaml-cpp_FOUND)
     unset(yaml_cpp_include_dir CACHE)
     find_path(yaml_cpp_include_dir yaml-cpp/yaml.h)
   endif()
-  set_target_properties(yaml-cpp PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${yaml_cpp_include_dir}")
+  set_target_properties(yaml-cpp::yaml-cpp PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${yaml_cpp_include_dir}")
 else()
   if(TARGET yaml-cpp)
     return()
