--- qt-creator-opensource-src-3.6.0/qtcreator.pri.libexec	2016-02-14 13:10:37.151868039 +0100
+++ qt-creator-opensource-src-3.6.0/qtcreator.pri	2016-02-14 13:12:02.182955481 +0100
@@ -108,7 +108,7 @@ osx {
     win32: \
         IDE_LIBEXEC_PATH = $$IDE_OUTPUT_PATH/bin
     else: \
-        IDE_LIBEXEC_PATH = $$IDE_OUTPUT_PATH/libexec/qtcreator
+        IDE_LIBEXEC_PATH = $$IDE_LIBRARY_PATH
     !isEqual(IDE_SOURCE_TREE, $$IDE_OUTPUT_PATH):copydata = 1
 
     LINK_LIBRARY_PATH = $$IDE_BUILD_TREE/$$IDE_LIBRARY_BASENAME/qtcreator
@@ -119,7 +119,7 @@ osx {
     win32: \
         INSTALL_LIBEXEC_PATH = $$QTC_PREFIX/bin
     else: \
-        INSTALL_LIBEXEC_PATH = $$QTC_PREFIX/libexec/qtcreator
+        INSTALL_LIBEXEC_PATH = $$QTC_PREFIX/$$IDE_LIBRARY_BASENAME/qtcreator
     INSTALL_DATA_PATH    = $$QTC_PREFIX/share/qtcreator
     INSTALL_DOC_PATH     = $$QTC_PREFIX/share/doc/qtcreator
     INSTALL_BIN_PATH     = $$QTC_PREFIX/bin
