--- qt-creator-opensource-src-4.8.2/src/libs/botan/botan.pro~	2019-05-11 12:45:09.000000000 +0200
+++ qt-creator-opensource-src-4.8.2/src/libs/botan/botan.pro	2019-05-11 12:45:26.217492890 +0200
@@ -16,8 +16,7 @@
 msvc: BOTAN_CC_TYPE = msvc
 else: clang: BOTAN_CC_TYPE = clang
 else: BOTAN_CC_TYPE = gcc
-contains(QT_ARCH, i386): BOTAN_ARCH_SWITCH = "--cpu=x86"
-else: contains(QT_ARCH, x86_64): BOTAN_ARCH_SWITCH = "--cpu=x86_64"
+BOTAN_ARCH_SWITCH = "--cpu=x32"
 BOTAN_MODULES = aes aes_ssse3 auto_rng bigint block cbc ctr des dh dsa ec_group ecdh ecdsa entropy \
                 filters hmac mode_pad pubkey rsa sha1 sha1_sse2 sha1_x86 sha2_32 sha2_32_x86 \
                 sha2_64 simd system_rng,emsa_pkcs1,pbes2,pbkdf2
